scripts/scripts.js

```diff
@@ -483,6 +483,19 @@ function decorateMarketo(main) {
   });
 }
 
+export function addTitleToLinks(el) {
+  el.querySelectorAll('a').forEach((a) => {
+    const ariaLabel = a.getAttribute('aria-label');
```

Comment:
Google uses title in place of aria-label if there is not one.

Reply:
I would prefer to have `decorateButtons` not set the `title` attribute but instead use `aria-label` when the link does not already contain `textContent`.

Reply:
After thinking on it, I think re-using the icon name for the parent link's aria-label may be the best option here. 
For linked text, we are covered (since it is text).